connector/failoverconnector/config.go

```diff
@@ -0,0 +1,48 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package failoverconnector // import "failoverconnector"
```

Comment:
I think this should be 
// import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/failoverconnector"